streamwebs_frontend/streamwebs/tests/models/test_site_model.py

```diff
@@ -0,0 +1,44 @@
+from django.test import TestCase
+
+from streamwebs.models import Site
+from django.contrib.gis.db import models
+
+
+class SiteTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = {
+            'name': models.TextField,
+            'site_type': models.TextField,
+            'description': models.TextField,
+            'location': models.PointField,
+            'created': models.DateTimeField,
+            'modified': models.DateTimeField,
+            'id': models.AutoField
+        }
+
+        self.optional_fields = {
+            'description'
+        }
+
+    def test_fields_exist(self):
+        model = model.get_model('streamwebs', 'site')
+        for field, field_type in self.expected_fields.items():
+            self.assertEqual(
+                field_type, type(model._meta.get_field_by_name(field[0])))
+
+    def test_no_extra_fields(self):
+        fields = Site._meta.get_all_field_names()
+        self.assertEqual(sorted(fields), sorted(self.expected_fields.keys()))
```

Comment:
You shouldn't have to sort python dicts when checking for equality. `assertEqual()` calls `assertDictEqual()` when passed two dicts.

See: https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertDictEqual